AmplifyPlugins/API/Tests/APIHostApp/AWSAPIPluginFunctionalTests/Base/TestConfigHelper.swift

```diff
@@ -6,16 +6,25 @@
 //
 
 import Foundation
-@testable import Amplify
+@_spi(InternalAmplifyConfiguration) @testable import Amplify
 
 class TestConfigHelper {
 
+    static var useGen2Configuration: Bool {
+        ProcessInfo.processInfo.arguments.contains("GEN2")
+    }
+
     static func retrieveAmplifyConfiguration(forResource: String) throws -> AmplifyConfiguration {
 
         let data = try retrieve(forResource: forResource)
         return try AmplifyConfiguration.decodeAmplifyConfiguration(from: data)
     }
 
+    static func retrieveAmplifyConfigurationV2(forResource: String) throws -> AmplifyConfigurationV2 {
```

Comment:
this should be `AmplifyOutputsData`, i must have written this before we did a switch to AmplifyOutputsData and forgot to update the integ tests since they don't build with main package